crates/openid4vci/src/credential_issuer/mod.rs

```diff
@@ -292,10 +292,16 @@ pub struct PreEvaluateCredentialRequestResponse {
 }
 
 /// Return type of the [`CredentialIssuer::evaluate_credential_request`]
-#[derive(Debug, Serialize, PartialEq, Eq, Default)]
+#[derive(Debug, Serialize, PartialEq, Eq)]
 pub struct CredentialIssuerEvaluateRequestResponse {
     /// Proof of possession, wrapping [`ProofOfPossession`]
     pub proof_of_possession: Option<ProofOfPossession>,
+
+    /// Identifier of the recipient. This identifier can be used when sending the credential.
+    pub recipient_id: Option<String>,
```

Comment:
the `proof` will contain the proof of possession of the key material the issued Credential shall be bound to. That is not nesarily the recipient_id (people are always going crazy about naming things).

Should we call it subject_id? (as it will be used as the credentialSubject.id).


Also interesting:

```
kid refers to a DID URL which identifies a particular key in the DID Document that the Credential shall be bound to. MUST NOT be present if jwk or x5c is present.
```

Most often the credentialSubject is a did not a did url pointing to a specific key. What should we return here? There full url?

Reply:
I was thinking of just the DID as it can directly be used. The proof of posession should contain the material to check the signature so the URL is not needed anymore I think.